### PR TITLE
#5231 use fix download url for nifi registry

### PR DESCRIPTION
**Pull request by tayowonibi**
use fix download url for nifi registry

### CODE REVIEW

File: salt/data-pipeline/nifi-registry.sls [hunk 1]
```diff
@@ -4,7 +4,7 @@
 download-nifi-registry:
     file.managed:
         - name: /root/downloads/nifi-registry-0.4.0-bin.tar.gz
-        - source: https://www-eu.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
+        - source: http://archive.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
```
Comment (de-code) [inline salt/data-pipeline/nifi-registry.sls]: I didn't pay attention when copying the URL, my mistake. Maybe better to keep using https.

**Review by de-code — APPROVED** [review]

File: salt/data-pipeline/nifi-registry.sls [hunk 2]
```diff
@@ -4,7 +4,7 @@
 download-nifi-registry:
     file.managed:
         - name: /root/downloads/nifi-registry-0.4.0-bin.tar.gz
-        - source: https://www-eu.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
+        - source: https://archive.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
```
Comment (giorgiosironi) [inline salt/data-pipeline/nifi-registry.sls:7]: why it has been removed from the other mirror? Will it happen again on `archive.apache.org`?

Comment (giorgiosironi) [reply salt/data-pipeline/nifi-registry.sls:7]: `https://www-eu.apache.org/dist/` seems to be a backup mirror only:
> Please only use the backup mirrors to download KEYS, PGP signatures and hashes (SHA* etc) -- or if no other mirrors are working. -- https://www.apache.org/dyn/closer.cgi

Comment (giorgiosironi) [reply salt/data-pipeline/nifi-registry.sls:7]: https://archive.apache.org/dist/nifi/nifi-registry/ seems to have all the possible versions so I'm cool with it.

Comment (giorgiosironi) [reply salt/data-pipeline/nifi-registry.sls:7]: (https://apache.org/dist/nifi/nifi-registry/ would be bad instead)

**Comment by giorgiosironi** [comment]
Waiting for `pr-fresh` to be green, then can be merged.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**